src/main.js

```diff
@@ -27,29 +27,61 @@ var currentCover = Cover
 
 
 var randomCoverButton = document.querySelector(".random-cover-button");
+var hideHomeView = document.querySelector(".home-view");
+var showFormView = document.querySelector(".form-view");
 var currentTitle = document.querySelector('.cover-title');
-var currentImageSrc = document.querySelector('.cover-image').src = getRandomBookInfo(covers);
+var currentImage = document.querySelector('.cover-image');
 var currentTagLine1 = document.querySelector('.tagline-1');
 var currentTagLine2 = document.querySelector('.tagline-2');
+var makeCoverButton = document.querySelector(".make-new-button");
 
+window.addEventListener("click", clickHandler);
+window.addEventListener("load", randomizeCover);
+// randomCoverButton.addEventListener("click", changeButton);
+//hideHomeView.addEventListener("click", changeView)
+// makeCoverButton.addEventListener("click", showCustomCover)
 
-randomCoverButton.addEventListener("click", changeButton);
-function changeButton() {
-currentTitle.innerText = getRandomBookInfo(titles);
-//currentImageSrc.innerText = getRandomBookInfo(covers);
-currentImageSrc = document.querySelector('.cover-image').src = getRandomBookInfo(covers);
+//redo for hide element
+function hideElement(element) {
+  document.querySelector(`.${element}`).classList.add("hidden");
+}
 
-currentTagLine1.innerText = getRandomBookInfo(descriptors);
-currentTagLine2.innerText = getRandomBookInfo(descriptors);
+
+function clickHandler(event) {
+  if (event.target.classList.contains("make-new-button")){
+    showCustomCover()
+  } else if (event.target.classList.contains("random-cover-button")){
+    changeButton()
+  }
```

Comment:
Play around with this logic to see if you can eliminate mosts global event listeners and query selectors.